### PR TITLE
[python] v1 breaking changes

### LINKED ISSUE #332: Add native support for more frameworks
- [x] scikit-learn API
- [x] tensorflow
- [x] xgboost
- [x] ~pyspark mllib~ (no contributors)
- [x] ~torch~ (probably undoable since the model can be anything)
- [x] pycaret (these are `scikit-learn` compatible through `finalize_model(model_only=False)`)
- [x] h2o
- [x] catboost (there is no conversion from `Pool` to `numpy`/`pandas` when categoricals are involved)

### PR DESCRIPTION
**Pull request by hbaniecki**
closes #332 mentions #368 

todo: recompile notebooks, documentation, ema

### CODE REVIEW

File: python/dalex/dalex/__init__.py [hunk 1]
```diff
@@ -1,16 +1,22 @@
-# dx.Explainer
-# from dalex.datasets import load_*
 from . import datasets
 from ._explainer.object import Explainer
-from ._arena.object import Arena
+from .arena.object import Arena
 
-__version__ = '0.4.1'
+__version__ = '0.4.1.9000'
```
Comment (pbiecek) [inline python/dalex/dalex/__init__.py:5]: should it be 1.0 ?

File: python/dalex/NEWS.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
 ## Changelog
 
+v1.0.0 (draft)
+----------------------------------------------------------------
+
+#### breaking changes
+
+These are summed up in ([#368](https://github.com/ModelOriented/DALEX/issues/368)):
+
+* rename modules: `dataset_level` into `model_expanations`, `instance_level` into `predict_explanations`, `_arena` module into `arena`
```
Comment (jakwisn) [inline python/dalex/NEWS.md:10]: typo in model_exp**l**antions

File: python/dalex/dalex/__init__.py (hunk 1 above)
Comment (hbaniecki) [reply python/dalex/dalex/__init__.py]: sure, I foresee the date of 23/12/202

File: python/dalex/NEWS.md (hunk 2 above)
Comment (hbaniecki) [reply python/dalex/NEWS.md]: thanks

**Review by piotrpiatyszek — APPROVED** [review]

**Pull request merged**